TESTPLAN.md

```diff
@@ -0,0 +1,47 @@
+Manual testing of the GCloud Plugin for Eclipse.
+
+# Install the plugin
+
+1. Launch Eclipse 4.5.
+
+2. If necessary, remove the old version of the plugin.
```

Comment:
Ideally nothing special. Mostly this is to make sure we actually are using the new version we're testing and that the install works.